.github/workflows/ci.yml

```diff
@@ -176,7 +176,7 @@ jobs:
                 ./configure.py --cl-inc-dir="$TEST_ENV_ROOT/include" --cl-lib-dir="$TEST_ENV_ROOT/lib"
                 git add -f siteconf.py
 
-                prepare_downstream_build "$DOWNSTREAM_PROJECT"
+                prepare_downstream_build "https://github.com/inducer/$DOWNSTREAM_PROJECT.git"
```

Comment:
How come the old version no longer works?

Reply:
Because I didn't realize the `prepare_downstream_build` was actually used outside of `ci-support` and did the backwards compatibility stuff only in the `test_downstream` wrapper.

Does this break anywhere else?